docs/scipp-neutron/groupby.ipynb

```diff
@@ -165,7 +165,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We use `scipp.groupby` with the desired bins and apply a `flatten` operation on dimension `spectrum`.\n",
+    "We use `scipp.groupby` with the desired bins and apply a `concatenate` operation on dimension `spectrum`.\n",
     "This is the events-data equivalent to summing histograms:"
```

Comment:
Existing typo "This is the events-data equivalent " -> "This is the event data equivalent " can be dealt with in follow up.